libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/listener/MysqlModePLParserListener.java

```diff
@@ -159,4 +159,11 @@ public void enterSp_decl(Sp_declContext ctx) {
         varibale.setVarType(type);
         this.varibaleList.add(varibale);
     }
+
+    @Override
+    public void enterCall_sp_stmt(PLParser.Call_sp_stmtContext ctx) {
+        this.sqlType = SqlType.CALL;
+        this.dbObjectType = DBObjectType.OTHERS;
```

Comment:
why others? 
setting `PROCEDURE` may be better?

Reply:
the statement like `call functionName()` also enters this method, so use OTHERS is better

Reply:
use PROCEDURE